packages/runtime/src/utils/initStateAndMethod.ts

```diff
@@ -15,7 +15,10 @@ export function initStateAndMethod(
   stateManager: StateManagerInterface,
   components: RuntimeComponentSchema[]
 ) {
-  components.forEach(c => initSingleComponentState(registry, stateManager, c));
+  components.forEach(c => {
+    stateManager.initSet.add(c.id);
```

Comment:
Maybe it is better to move this line into `initSingleComponentState`?

Reply:
`initSingleComponentState` has been called in `UnmountImplWrapper` and `ImplWrapper`, while `initStateAndMethod` only called by the App and ModuleRenderer.

I'm not sure that scenarios need mark init set to true.